de.tumexam.cli.yml

```diff
@@ -0,0 +1,139 @@
+app-id: de.tumexam.cli
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: tumexam-cli
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=session-bus
```

Comment:
You should figure which DBus names are needed instead.

Reply:
Done. Replaced it `--own-name=de.tumexam`

Reply:
`de.tumexam.cli` is owned by default based in the app-id. I don't think `de.tumexam` is even a valid D-Bus name.

Reply:
For me it works since I define it here:
https://github.com/COM8/tumexam-cli/blob/ba0072e64048ab399671fd87e47017d51f33818b/src/ui/UiContext.cpp#L6

But I guess I should replace this one with `de.tumexam.cli`.

Reply:
Changed it.